roles/kube-node-common/defaults/main.yml

```diff
@@ -15,6 +15,14 @@ skip_kubelet_upgrade: False
 
 dependencies:
   # To pin dependencies for each Kubernetes version
+  "1.19.16":
+    critools_version: "1.25.0"
+  "1.20.15":
+    critools_version: "1.25.0"
+  "1.21.14":
+    critools_version: "1.25.0"
+  "1.22.13":
+    critools_version: "1.25.0"
```

Comment:
just curious, is critools backwards compatible like that? I could not find anything confirming it in a quick look a their repo

Reply:
Yes, it should, the compatibility `should` be 1:1 but there is no 1:1 on package managers, and it should not be a problem (the only problem we had was between 1.24 and 1.25, i think)